### ISSUE TITLE
Define model 1 CV and Monitoring strategy

### ISSUE THREAD

**Issue by RaymondBrien** [RaymondBrien/cherry-ml#21]
Training options to consider:
- batch_size: try [16, 32, 64, 128]
- learning_rate scheduling: reduce_lr_on_plateau or cosine decay
- early stopping patience: try [5, 10, 15] epochs
- validation_split: try [0.1, 0.2, 0.3]
- class_weights for imbalanced dataset

Example usage with different input sizes:

Smaller input:
model = create_powdery_mildew_classifier(input_shape=(160, 160, 3))

Larger input:
model = create_powdery_mildew_classifier(input_shape=(299, 299, 3))   

Systematic Tuning Approach:

Start with architecture modifications (layer sizes, depth)
Then tune regularization parameters (dropout, batch norm)
Finally adjust optimization parameters (learning rate, batch size)

Cross-Validation Strategy:
pythonCopyfrom sklearn.model_selection import KFold
k_fold = KFold(n_splits=5, shuffle=True)
# Use this to validate hyperparameter choices

Monitoring Tips:

Watch validation loss for overfitting
Monitor precision/recall trade-offs
Use TensorBoard for visualization of metrics
